kaadivvvv.user.js

```diff
@@ -59,4 +57,4 @@ window.sessionStorage.logoPlayed = true;
 * A colui che sta analizzando questo script per commessa, se posso darti del tu, ti ringrazio per le sfide che avanzi.
 * Anche se la programmazione web non e il mio forte, chissa che magari un giorno ci troveremo seduti intorno ad un tavolo
 * a parlare di algoritmi e bere in compagnia :) E se sei una lei, tanto di cappello!
-*/
+*/
```

Comment:
```suggestion
*/

```
Meglio lasciarlo per evitare commit dove cambia solo il ritorno a capo

Reply:
Ops, l'ho persa in qualche momento 😅